ts/features/wallet/creditCard/screen/CreditCardDetailScreen.tsx

```diff
@@ -1,38 +1,45 @@
 import * as React from "react";
 
-import { connect } from "react-redux";
+import { sequenceS } from "fp-ts/lib/Apply";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
 import LoadingSpinnerOverlay from "../../../../components/LoadingSpinnerOverlay";
 import WorkunitGenericFailure from "../../../../components/error/WorkunitGenericFailure";
+import { PaymentCardBig } from "../../../../components/ui/cards/payment/PaymentCardBig";
 import { IOStackNavigationRouteProps } from "../../../../navigation/params/AppParamsList";
 import { WalletParamsList } from "../../../../navigation/params/WalletParamsList";
-import { GlobalState } from "../../../../store/reducers/types";
+import { useIOSelector } from "../../../../store/hooks";
 import { creditCardByIdSelector } from "../../../../store/reducers/wallet/wallets";
 import { CreditCardPaymentMethod } from "../../../../types/pagopa";
 import { idPayAreInitiativesFromInstrumentLoadingSelector } from "../../../idpay/wallet/store/reducers";
 import BasePaymentMethodScreen from "../../common/BasePaymentMethodScreen";
 import PaymentMethodFeatures from "../../component/features/PaymentMethodFeatures";
-import CreditCardComponent from "../component/CreditCardComponent";
+import { IOLogoPaymentType } from "../../../../components/core/logos";
 
 export type CreditCardDetailScreenNavigationParams = Readonly<{
   // Since we don't have a typed ID for the payment methods, we keep the creditCard as param even if it is then read by the store
   creditCard: CreditCardPaymentMethod;
 }>;
 
-type Props = ReturnType<typeof mapStateToProps> &
-  IOStackNavigationRouteProps<WalletParamsList, "WALLET_CREDIT_CARD_DETAIL">;
+type Props = IOStackNavigationRouteProps<
+  WalletParamsList,
+  "WALLET_CREDIT_CARD_DETAIL"
+>;
 
 /**
  * Detail screen for a credit card
- * @constructor
  */
-const CreditCardDetailScreen: React.FunctionComponent<Props> = props => {
+const CreditCardDetailScreen = ({ route }: Props) => {
   const [walletExisted, setWalletExisted] = React.useState(false);
-  const paramCreditCard: CreditCardPaymentMethod =
-    props.route.params.creditCard;
-  const { areIdpayInitiativesLoading } = props;
+  const paramCreditCard: CreditCardPaymentMethod = route.params.creditCard;
   // We need to read the card from the store to receive the updates
   // TODO: to avoid this we need a store refactoring for the wallet section (all the component should receive the id and not the wallet, in order to update when needed)
-  const storeCreditCard = props.creditCardById(paramCreditCard.idWallet);
+  const storeCreditCard = useIOSelector(state =>
+    creditCardByIdSelector(state, paramCreditCard.idWallet)
+  );
+  const areIdpayInitiativesLoading = useIOSelector(
+    idPayAreInitiativesFromInstrumentLoadingSelector
+  );
```

Comment:
This selector seems to produce a "graphical glitch" only into a CreditCard detail screen showing a loading indicator for a while as showed in the following video:

https://github.com/pagopa/io-app/assets/34343582/d088bb61-630d-4aba-ae27-b6d1cf3d7108

Reply:
this is a weird one, it's an old implementation and nothing was touched; could it be something with a failing and retrying fetch? 

theoretically that loading should trigger only during the screen's  first render since all subsequent calls are handled with a "refresh" boolean prop that does not set the object to a loading state.

I think we should check this in a pair session or anyways aside from this PR since I have a good degree of certainty that this PR is not responsible of this glitch.